subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/bintray/BintrayReleasePluginTest.groovy

```diff
@@ -1,17 +1,45 @@
 package org.shipkit.internal.gradle.bintray
 
-import org.gradle.testfixtures.ProjectBuilder
+import org.shipkit.gradle.notes.UpdateReleaseNotesTask
 import org.shipkit.internal.gradle.java.JavaBintrayPlugin
-import spock.lang.Specification
+import org.shipkit.internal.gradle.notes.ReleaseNotesPlugin
+import testutil.PluginSpecification
 
-class BintrayReleasePluginTest extends Specification {
+class BintrayReleasePluginTest extends PluginSpecification {
 
-    def project = new ProjectBuilder().build()
+    def "configures tasks with user pulication repo"() {
+        conf.releaseNotes.publicationRepository = "publicRepo"
 
-    def "applies"() {
-        expect:
         project.plugins.apply(BintrayReleasePlugin)
         project.plugins.apply(JavaBintrayPlugin)
+
+        project.bintray.pkg.userOrg = "some-org"
+        project.bintray.pkg.repo = "some-repo"
+        project.bintray.pkg.name = "some-pkg"
```

Comment:
Configuration of bintray extension seems irrelevant for this test. UpdateReleaseNotesTask will use conf.releaseNotes.publicationRepository to set the value of publicationRepository.